src/solidity-frontend/solidity_convert.h

```diff
@@ -188,6 +168,38 @@ class solidity_convertert
   void convert_type_expr(const namespacet &ns, exprt &dest, const typet &type);
   bool convert_hex_literal(std::string the_value, exprt &dest, const int n = 0);
 
+  contextt &context;
+  namespacet ns;
+  nlohmann::json
+    &ast_json; // json for Solidity AST. Use vector for multiple contracts
```

Comment:
Please put the comment above the declaration, so it isn't broken across lines.

Reply:
Changed as suggested, thanks